ui/src/app/components/with-leo-cookie.spec.tsx

```diff
@@ -153,15 +147,14 @@ describe(withLeoCookie.name, () => {
       .spyOn(leoProxyApi(), 'setCookie')
       .mockResolvedValue(null);
 
-    const wrapper = createWrapper();
+    const { unmount } = createWrapper();
 
     expect(setCookieStub).toHaveBeenCalledTimes(1);
 
-    wrapper.unmount();
+    unmount();
```

Comment:
I could be wrong...but looks like it needs to be `unmount.unmount()`?

Reply:
I don't know much about what it's doing here, but why do you think that?

Reply:
I think this looks good. It looks like Joel is destructuring the unmount function from the return of render.

A little more detail about that can be found here:
https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-wrapper-as-the-variable-name-for-the-return-value-from-render